ios/Podfile

```diff
@@ -49,7 +33,8 @@ target 'ReactNativeStarter' do
     react_native_post_install(
       installer,
       config[:reactNativePath],
-      :mac_catalyst_enabled => false
+      :mac_catalyst_enabled => false,
+      # :ccache_enabled => true
```

Comment:
comment

Reply:
This line should be commented out according to the upgrade guide ->
<img width="964" alt="image" src="https://github.com/TheSoftwareHouse/react-native-starter-boilerplate/assets/39768412/57742bb0-d125-4fbd-9974-0015f6cdcb03">